selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/PythonFileEditingTest.java

```diff
@@ -132,42 +112,34 @@ public void checkAutocompleteFeature() throws Exception {
     editor.enterAutocompleteProposal("function() ");
     editor.waitTextIntoEditor("print(object.function");
     editor.typeTextIntoEditor("())");
```

Comment:
What is the propose of typing text `"())"` into editor if we don't check anything after that?